src/scss/components/dropdown-select/mixins/SelectFacade.scss

```diff
@@ -7,7 +7,7 @@
   }
   @else {
     border: 1px solid $gray-500;
```

Comment:
please change to border: 1px solid var(--hxDropdown-border, $gray-500);

Reply:
Changes have been done, please review.